extlibs/vili/CMakeLists.txt

```diff
@@ -28,7 +28,14 @@ set(VILI_SOURCES
 add_library(vili ${VILI_HEADERS} ${VILI_SOURCES})
 
 target_link_libraries(vili fifo_map)
-target_link_libraries(vili fmt)
+
+if(OBE_USE_VCPKG)
+    find_package(fmt CONFIG REQUIRED)

```

Comment:
Do you really need to do a `find_package` again ?

Reply:
Yes, you need it there unfortunately. A wave of errors followed when that wasn't set. I can try to remove that from the Core CmakeList

Reply:
Since `fmt` is required by both vili and ÖbEngineCore, I think the `find_package` should be located in the root `CMakeLists.txt` ?